.github/workflows/ci.yml

```diff
@@ -56,8 +44,9 @@ jobs:
       KRATOS_BUILD_TYPE: ${{ matrix.build-type }}
       KRATOS_CI_CHANGED_FILES: ${{needs.changed-files.outputs.files}}
       KRATOS_CI_APPLICATIONS: ".github/workflows/ci_apps_linux.json"
-      OMPI_MCA_rmaps_base_oversubscribe: 1 # Allow oversubscription for MPI (needed for OpenMPI >= 3.0)
-      OMPI_MCA_btl_vader_single_copy_mechanism: none # suppressing some annoying OpenMPI messages
+      KRATOS_CI_CORES: 4                              # Current limits are 4 CPU and 16 GB Ram
```

Comment:
so far we used 2, can you briefly explain why we can use 4 now?

Reply:
It is what they offer now:

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories